hostutils/psu.md

```diff
@@ -1,9 +1,116 @@
-# Phoenix Software Uploader (psu)
+# Phoenix Serial Uploader (psu)
 
-Documentation under development.
+`psu` is a tool designed to load scripts onto Phoenix-RTOS devices using the SDP (Script Download Protocol) protocol.
+This tool allows users to execute various commands on Phoenix-RTOS devices by sending scripts containing
+specific instructions.
```

Comment:
We can also mention that it is used for NXP boards and that we are using it in the flashing process for targets like `imxrt106x` or `imxrt117x`.